<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>无人机吊舱地面站V1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/file/image/Paper_airplane_24px_1181455_easyicon.net.ico</normaloff>:/file/image/Paper_airplane_24px_1181455_easyicon.net.ico</iconset>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>901</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideRight</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_settings">
     <attribute name="title">
      <string>串口设置</string>
     </attribute>
     <attribute name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;111&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>401</width>
        <height>241</height>
       </rect>
      </property>
      <property name="title">
       <string>接收区</string>
      </property>
      <widget class="QTextBrowser" name="ReceivedText">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>351</width>
         <height>181</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>270</y>
        <width>401</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>发送区：</string>
      </property>
      <widget class="QLineEdit" name="SendText">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>361</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="SendButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>244</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>串口参数</string>
      </property>
      <widget class="QComboBox" name="comboBox_portName">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_Bandrate">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
      </widget>
      <widget class="QComboBox" name="comboBox_Stopbit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>32</y>
         <width>52</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>端口号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>81</y>
         <width>39</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>130</y>
         <width>39</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>42</x>
         <y>179</y>
         <width>39</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_databit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>180</y>
         <width>62</width>
         <height>21</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>云台状态</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>横滚</string>
      </property>
      <widget class="QTextEdit" name="Roll_Display">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Pitch_Display">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextEdit" name="Yaw_Display">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>541</width>
        <height>321</height>
       </rect>
      </property>
      <property name="title">
       <string>角度</string>
      </property>
      <widget class="QwtDial" name="Dial_Roll" native="true">
       <property name="geometry">
        <rect>
         <x>36</x>
         <y>22</y>
         <width>150</width>
         <height>280</height>
        </rect>
       </property>
       <property name="lineWidth" stdset="0">
        <number>4</number>
       </property>
      </widget>
      <widget class="QwtDial" name="Dial_Pitch" native="true">
       <property name="geometry">
        <rect>
         <x>206</x>
         <y>22</y>
         <width>150</width>
         <height>280</height>
        </rect>
       </property>
       <property name="lineWidth" stdset="0">
        <number>4</number>
       </property>
      </widget>
      <widget class="QwtDial" name="Dial_Yaw" native="true">
       <property name="geometry">
        <rect>
         <x>370</x>
         <y>22</y>
         <width>150</width>
         <height>280</height>
        </rect>
       </property>
       <property name="lineWidth" stdset="0">
        <number>4</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>39</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>60</y>
         <width>46</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>60</y>
         <width>41</width>
         <height>21</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>100</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>311</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QwtThermo" name="Thermo_Roll" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>200</width>
         <height>41</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">backgroundcolor:rgb(255, 0, 0)</string>
       </property>
       <property name="alarmEnabled" stdset="0">
        <bool>true</bool>
       </property>
       <property name="pipeWidth" stdset="0">
        <number>15</number>
       </property>
      </widget>
      <widget class="QwtThermo" name="Thermo_Pitch" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>200</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">backgroundcolor:rgb(0, 0, 255)</string>
       </property>
       <property name="alarmEnabled" stdset="0">
        <bool>true</bool>
       </property>
       <property name="pipeWidth" stdset="0">
        <number>15</number>
       </property>
      </widget>
      <widget class="QwtThermo" name="Thermo_Yaw" native="true">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>110</y>
         <width>200</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">backgroundcolor:rgb(0, 255, 127)</string>
       </property>
       <property name="alarmEnabled" stdset="0">
        <bool>true</bool>
       </property>
       <property name="pipeWidth" stdset="0">
        <number>15</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Roll</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>30</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>65536</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_13">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>70</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>65536</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>110</y>
         <width>54</width>
         <height>12</height>
        </rect>
       </property>
       <property name="text">
        <string>65536</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_20">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>41</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string>Yaw</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>波形显示</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>851</width>
        <height>451</height>
       </rect>
      </property>
      <property name="title">
       <string>三轴角度</string>
      </property>
      <widget class="QwtPlot" name="qwtPlot">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>801</width>
         <height>331</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="OpenserialPort">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CloseSerialPort">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭串口</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtThermo</class>
   <extends>QWidget</extends>
   <header>qwt_thermo.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
